### DISCUSSION TITLE
QueryExecMode for use with PgBouncer which has transaction pool mode

### DISCUSSION THREAD

**Discussion by stasvo** [jackc/pgx#1531] [Q&A] [answered]
In description I see that in case of use pgBouncer need to select one of these modes: QueryExecModeExec or QueryExecModeSimpleProtocol. These also information: prefer QueryExecModeExec over QueryExecModeSimpleProtocol whenever possible.
My question: is it possible to use QueryExecModeExec in case of use PgBouncer in transaction pool mode or in this case exist only option to use QueryExecModeSimpleProtocol?

**Comment by jackc** [comment, marked as answer]
AFAIK, either should work.

**Reply by stasvo** [reply]
Thanks for answer.
